src/components/text-editor/text-editor.tsx

```diff
@@ -111,6 +112,16 @@ export class TextEditor implements FormComponent<string> {
     @Prop({ reflect: true })
     public allowResize: boolean = true;
 
+    /**
+     * Defines how the editor should look
+     * - `standard`: The default look of the editor
+     * - `minimal`: A more compact look of the editor, useful for
+     * displaying on mobile devices for instance space is limited.
+     * It will remove the toolbar, unless the editor is in focus.
+     */
```

Comment:
```suggestion
    /**
     * Specifies the visual appearance of the editor.
     *
     * - `standard`: The default editor appearance with a full toolbar and
     *    standard layout.
     * - `minimal`: A compact editor appearance, ideal for limited space
     *    scenarios such as mobile devices. In this mode, the toolbar is hidden
     *    until the editor is focused.
     */
```

Reply:
I mainly wanted to fix the grammatical issue in "useful for displaying on mobile devices for instance space is limited", but I liked the overall improvements ChatGPT suggested 😊

Reply:
nice. added it as a fixup